docs/pages/demo/datetime-picker/index.mdx

```diff
@@ -32,9 +32,10 @@ DateTimePicker is a complex control, so you may consider using `MobileDateTimePi
 
 #### Date & Time validation
 
-It is possible to restrict date and time selection in couple of different ways. If you need to restrict time selection
-you could use `minDateTime` to restrict selecting date and time before particular moment of time.
-Or you can use `minTime` to disable a part of time in each day.
+It is possible to restrict date and time selection in two ways:
+
+* using `minDateTime`, you can restrict time selection before a particular monet in time
```

Comment:
```suggestion
* using `minDateTime`, you can restrict time selection before a particular moment of time
```

Reply:
Why doesn't GitHub have spell checking in their CodeMirror instance :-/